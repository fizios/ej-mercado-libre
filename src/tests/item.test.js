import React from "react";
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import listMock from './mockJson/list.json';

import Item from '../components/item';

const mockProps = listMock.items[0];

const element = (
  <MemoryRouter>
    <Item {...mockProps} />
  </MemoryRouter>
);

describe('[ ITEM ]', () => {
  it('Renders and it matches snapshot', () => {
    const wrapper = renderer.create(element);
    expect(wrapper).toMatchSnapshot();
  });
});
