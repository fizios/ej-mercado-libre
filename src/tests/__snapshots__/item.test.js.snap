// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ ITEM ] Renders and it matches snapshot 1`] = `
<div
  className="item-container"
  onClick={[Function]}
>
  <div
    className="item-data"
  >
    <div
      className="picture"
    >
      <img
        alt="Micrófono Fifine F-800 Condensador Negro"
        src="http://http2.mlstatic.com/D_847685-MLA32553351853_102019-I.jpg"
      />
    </div>
    <div
      className="info-container"
    >
      <p
        className="price"
      >
        ARS
         
        3970
        <img
          alt="free_shipping"
          src="/icons/shipping.png"
        />
      </p>
      <p
        className="title"
      >
        Micrófono Fifine F-800 Condensador Negro
      </p>
    </div>
  </div>
  <div
    className="item-location"
  >
    Capital Federal
  </div>
</div>
`;
